include/ccf/node/cose_signatures_config.h

```diff
@@ -6,13 +6,16 @@
 
 #include <string>
 
-struct COSESignaturesConfig
```

Comment:
This was deliberately not namespaced, because it started under startup_config.h, out of shallow consistency with other config types. I do not remember why we don't namespace those, and the right thing to do is perhaps to namespace them all.

Reply:
Huh, I'd missed that we still had anything un-namespaced in the public headers. I agree the right approach is to namespace them all, and I'll look at automated enforcement too.